indexes/plugin.js

```diff
@@ -143,11 +143,11 @@ module.exports = class Plugin {
     const subClassReset = this.reset
     this.reset = (cb) => {
       if (subClassReset) subClassReset.call(this)
+      this.batch = []
+      this.offset.set(-1)
```

Comment:
Oh, I didn't see that one. I *think* they *should* be different because line 137 happens when `this.offset` obz is still undefined. `this.offset` at undefined is not so harmful, because it never `=== log.since.value` which is many times checked in `db.js` ("core.js" nowadays)

Reply:
right, it is also not the problem here, was just wondering